pycaw/tools/app_mixer/launch.py

```diff
@@ -0,0 +1,88 @@
+# ____ COM WITH MULTITHREADED APARTMENT ____
+import sys
+
+sys.coinit_flags = 0  # noqa: E402
+
+# _____ ADVANCED WINDOWS COMMUNICATION _____
+from comtypes import COMError
+
+# _________________ PYCAW _________________
+from pycaw.api.audiopolicy import IAudioSessionControl2
+
+# ______________ GUI TKINTER ______________
+from pycaw.tools.app_mixer.app_controll_and_gui import AppRow, RootFrame
+
+# ________________Register_______________
+from pycaw.tools.app_mixer.callbacks_and_manager import SessionManager
+from pycaw.utils import AudioSession, AudioUtilities
+
+
+class StartUp:
+    """
+    Register new session callback.
+    Scan for running session and add them to session_manager
+    """
+    @classmethod
+    def __init__(cls, RootFrameInstance, AppRowClass):
+
+        try:
+            mgr = AudioUtilities.GetAudioSessionManager()
+        except COMError:
+            exit("No speaker connected")
+
+        session_manager = SessionManager(RootFrameInstance, AppRowClass)
+        mgr.RegisterSessionNotification(session_manager)
+
+        # has to get called !!!
+        # to make IAudioSessionNotification::OnSessionCreated working
+        sessionEnumerator = mgr.GetSessionEnumerator()
+
+        # get all active sessions
+        count = sessionEnumerator.GetCount()
+        all_sessions = [
+            cls._get_sessions(i, sessionEnumerator) for i in range(count)
+        ]
+
+        # add sessions to session_manager
+        [session_manager.add_session(session) for session in all_sessions]
```

Comment:
I'd rather make it a tuple (immutable) specially if it's saved as a member
```suggestion
        tuple(session_manager.add_session(session) for session in all_sessions)
```

Reply:
👍
... but just out of curiosity: what do you mean with "if it's saved as a member"

Reply:
My bad I miss read that line somehow sorry :facepalm: 
I read it as something like:
```python
session_manager = [session_manager.add_session(session) for session in all_sessions]
cls.session_manager = session_manager
```
Not sure how I brainfarted like this. You'were just doing a `add_session()` via a list comprehension.
Yeah you can revert it doesn't make sense what I wrote and it makes things confusing

Reply:
okay ;) no problem ;)